include/jemalloc/internal/tsd.h

```diff
@@ -87,6 +87,8 @@ typedef void (*test_callback_t)(int *);
     O(tcache_gc_event_wait,	uint64_t,		uint64_t)	\
     O(prof_sample_event_wait,	uint64_t,		uint64_t)	\
     O(prof_sample_last_event,	uint64_t,		uint64_t)	\
+    O(stats_interval_event_wait,	uint64_t,	uint64_t)	\
+    O(stats_interval_last_event,	uint64_t,	uint64_t)	\
```

Comment:
That's right. However it's possible to optimize this away, since this has a fixed interval (versus, the prof sample which has randomness in it). I plan to give it a try when working on the thread event on dealloc.

Reply:
Right, if you don't want to be precise: your `last_event - last_stats_event` may be more than the batch size, so if you want to accumulate the precise amount to the global counter, you might still need `stats_interval_last_event`.